docs/docs/user-guide/qa.md

```diff
@@ -0,0 +1,84 @@
+# Quality Assurance
+
+## Overview
+
+Browsertrix Crawler has the capability to "re-crawl" an existing crawl to compare what the browser encountered on a website during crawling against the replay of the crawl WACZ. The WACZ produced by this analysis run includes additional comparison data (stored as WARC `resource` records) for the pages found during crawling against their replay in ReplayWeb.page along several dimensions, including screenshot, extracted text, and page resource comparisons.
+
+!!! note
+
+    QA features described on this page are available in Browsertrix Crawler releases 1.1.0 and later.
```

Comment:
I guess these are good because we don't version our docs?  No harm in them really, though as we add new features and reference them on pages like this, they run the risk of going out of date?

Reply:
Yeah, probably don't wanna go crazy with this but it's also worth flagging since this is a big feature that doesn't exist in older releases I think.

Reply:
You're right that better docs versioning would definitely help!